pkg/agent/controller/traceflow/packetin.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Should we move packet processing to a separate pkg instead of agent/controller/traceflow? Like agent/traceflow or agent/openflow/traceflow?
@wenyingd @tnqn

Reply:
I would prefer package "agent/openflow/...". I believe we might have other features need to process PacketIn messages.

Reply:
Moved to pkg/agent/openflow/